Sources/Tasks/Utf8EntityDecode.php

```diff
@@ -193,6 +193,30 @@ protected function decode(string $string): string
 		return str_contains($string, '&') ? mb_decode_numericentity($string, [0x010000, 0x10FFFF, 0, 0xFFFFFF], 'UTF-8') : $string;
 	}
 
+	/**
+	 * Adds a new instance of this task to the task list.
+	 */
+	protected function respawn(array $details, int $time = 0): void
+	{
+		Db::$db->insert(
+			method: 'insert',
+			table: '{db_prefix}background_tasks',
+			columns: [
+				'task_class' => 'string-255',
+				'task_data' => 'string',
+				'claimed_time' => 'int',
+			],
+			data: [
+				[
+					get_class($this),
+					json_encode($this->_details),
+					0,
+				],
+			],
+			keys: [],
+		);
+	}
+
```

Comment:
This doesn't need to be added here. We can just use the inherited method from the base class. Instead, the only change we need to make is to change `$this->respawn();` to `$this->respawn($this->_details);` on line 166 above.

Reply:
That's what I get for not checking the reason and just changing the visibility to resolve the error.